plasmapy/formulary/collisions/frequencies.py

```diff
@@ -25,125 +24,144 @@
+    r"""
+    Compute collision frequencies between test particles (labeled 'a')
+    and field particles (labeled 'b').
 
-        Parameters
-        ----------
-        test_particle : `~plasmapy.particles.ParticleLike`
-            The test particle streaming through a background of field
-            particles.
+    Parameters
+    ----------
+    test_particle : |particle-like|
+        The test particle streaming through a background of field
+        particles.
 
-        field_particle : `~plasmapy.particles.ParticleLike`
-            The background particle being interacted with.
+    field_particle : |particle-like|
+        The background particle that ``test_particle`` is interacting
+        with.
 
-        v_drift : `~astropy.units.Quantity`
-            The relative drift between the test and field particles.
-            Cannot be negative.
+    v_drift : `~astropy.units.Quantity`, |keyword-only|
+        The relative drift velocity between the test and field
+        particles. Cannot be negative.
 
-        T_b : `~astropy.units.Quantity`
-            The temperature of the background field particles in units
-            convertible to degrees Kelvin.
+    T_b : `~astropy.units.Quantity`, |keyword-only|
+        The temperature of the background field particles in units
+        convertible to kelvin or eV per particle.
 
-        n_b : `~astropy.units.Quantity`
-            The number density of the background field particles in
-            units convertible to :math:`\frac{1}{m^{3}}`.
+    n_b : `~astropy.units.Quantity`, |keyword-only|
+        The number density of ``field_particle`` in units convertible to
+        m\ :sup:`-3`.
 
-        Coulomb_log : `~astropy.units.Quantity`
-            The value of the Coulomb logarithm for the interaction.
+    Coulomb_log : `~astropy.units.Quantity`, |keyword-only|
+        The value of the Coulomb logarithm for the interaction.
 
-        Raises
-        ------
-        `ValueError`
-            If the specified v_drift and n_b arrays don't have equal
-            size.
+    Raises
+    ------
+    `ValueError`
+        If the specified ``v_drift`` and ``n_b`` arrays do not have
+        equal size.
 
-        Notes
-        -----
-        The frequency of collisions between a test particle (subscript
-        :math:`\alpha`) and a field particle (subscript :math:`\beta`)
-        each with mass  :math:`m` and charge :math:`e` are given by
-        four differential equations:
+    Notes
+    -----
+    The frequency of collisions between a test particle (subscript
+    :math:`α`) and a field particle (subscript :math:`β`) each with mass
+    :math:`m` and charge :math:`e` are given by four differential
+    equations:
 
-            momentum loss: :math:`\frac{d\textbf{v}_{α}}{dt}=-ν_{s}^{α\backslashβ}\textbf{v}_{α}`
+        momentum loss: :math:`\frac{d\mathbf{v}_α}{dt}
+        = - ν_{s}^{α \backslash β} \mathbf{v}_α`
```

Comment:
For this and the follow equations, I still need to verify that they are rendering just like they did before.